src/control/server/testdata/certs/admin.crt

```diff
@@ -0,0 +1,114 @@
+Certificate:
```

Comment:
Won't these test certs expire eventually and need to be regenerated? Last week I was messing around a little bit trying to get Go unit tests to automatically generate their own certs and write them to files in a temp directory, then clean them after the test is done. It seems plausible for the Go tests at least.

Reply:
We could do that. It will expire in 3 years but if you have something to generate certs for the tests that would probably work better. The only thing is you now have the openssl tool as a requirement for running the tests.

Reply:
We can actually generate them within Go using the crypto libraries, no need to shell out to openssl. I can show you what I have - maybe it would be helpful to you.